scripts/pipelines/build.vms.sh

```diff
@@ -192,9 +192,6 @@ write_vm_startup_script() {
 	local script_destination_path=$2
 	local primary_disk=$3
 
-	local vm_uuid=$(uuidgen)
```

Comment:
Is there a Dockerfile that also has to be updated?

Reply:
There is not. I found this issue when running the script on my xidax machine - which doesn't have the binary. Part of the wrongness of this line is that it doesn't use the `$PYREX_RUN` wrapper, and is trying to run the command on the host machine. So it breaks on any machine without the binary.